.github/workflows/build-and-push.yml

```
@@ -53,8 +53,22 @@ jobs:
             _tag=${{ matrix.tag }}
           fi
           echo "tag=${_tag}" >> $GITHUB_OUTPUT
+          
+      - name: Check if dockerfile is c9s , then build & push s2i-core for multi-arch
+        if: matrix.dockerfile == 'Dockerfile.c9s'
+        uses: sclorg/build-and-push-action@v4
+        with:
+          registry: "quay.io"
+          registry_namespace: ${{ matrix.registry_namespace }}
+          registry_username: ${{ secrets[matrix.quayio_username] }}
+          registry_token: ${{ secrets[matrix.quayio_token] }}
+          dockerfile: "core/${{ matrix.dockerfile }}"
+          image_name: "s2i-core${{ matrix.suffix }}"
+          tag: ${{ steps.tag.outputs.tag }}
+          archs: amd64, ppc64le, s390x
```

What if we specify only `ppc64le, s390x` here and build the `amd64` with during next step with other images? Could that work?
```suggestion
          archs: ppc64le, s390x
```

---

let me check , will update you post testing

---

As conveyed , it didn't work that way . I have to add that old check if: matrix.dockerfile != 'Dockerfile.c9s' as well , otherwise it will override the tag again for c9s 
https://github.com/modassarrana89/s2i-base-container/actions/runs/5042239798/jobs/9042671876